PaddleCV/object_detection/ppdet/data/data_feed.py

```diff
@@ -41,7 +41,7 @@
 ]
 
 
-def create_reader(feed, max_iter=0):
+def create_reader(feed, max_iter=0, image_or_dir=None, image_ext=None):
```

Comment:
Dot no add image_or_dir, image_ext here.

Why not by merging cli args  into configs?

Reply:
Done